pkg/middleware/request_metrics.go

```diff
@@ -59,11 +60,15 @@ func RequestMetrics(features featuremgmt.FeatureToggles) web.Handler {
 		c.Next()
 
 		handler := "unknown"
-
 		if routeOperation, exists := RouteOperationNameFromContext(c.Req.Context()); exists {
 			handler = routeOperation
 		}
 
+		if features.IsEnabled(featuremgmt.FlagLogRequestsInstrumentedAsUnknown) {
```

Comment:
should this go to `else` branch above if routeOperation not found?

Reply:
:+1: